### ISSUE TITLE
UI Test failing when running on an app with RestKit cocoa pod installed

### ISSUE THREAD

**Issue by Claus79** [CocoaPods/CocoaPods#4609]
I'm creating my first UI Test for my iOS app. Every time I run the test I get the error:

`UI Testing Failure - App accessibility isn't loaded
(please see attached image)`

plus an error message in the debug console:

`dyld: Library not loaded: @rpath/RestKit.framework/RestKit
  Referenced from: /Users/Claus/Library/Developer/CoreSimulator/Devices/<code>/data/Containers/Bundle/Application/<code>/MyApp.app/MyApp
  Reason: image not found`

I would like to point out that the RestKit (installed using CocoaPods) library doesn't give any problem when I build and run my app on the simulator or real device.

**Comment by segiddins** [comment]
Do you have RestKit specified in your Podfile for the UI Test target?

**Comment by Claus79** [comment]
at the moment I have no targets specified in the pod file

**Comment by segiddins** [comment]
that'd be the issue. If you want to use the pods in your UI Testing target, you need to specify that in your Podfile.

**Comment by Claus79** [comment]
Ok I created two targets in the Podfile with the same pods

when I run pod install I get this message:

CocoaPods did not set the base configuration of your project because your project already has a custom config set. In order for CocoaPods integration to work at all, please either set the base configurations of the target `MyApp` to `Pods/Target Support Files/Pods-MyApp/Pods-MyApp.debug.xcconfig` or include the `Pods/Target Support Files/Pods-MyApp/Pods-MyApp.debug.xcconfig` in your build configuration.

I checked my configuration and it's:

Project > Info > Configuration > Debug
MyApp -> Pods.debug
MyAppUITests -> Pods-MyAppUITests.debug

so it should be ok, but now the project doesn't build anymore

![screen shot 2015-11-30 at 17 13 29](https://cloud.githubusercontent.com/assets/1422605/11478583/cdbf3926-9785-11e5-96de-52fca7665769.png)

**Comment by segiddins** [comment]
Change `MyApp` to `Pods-MyApp.debug.xcconfig`.

**Comment by Claus79** [comment]
I have done it but now I have a new error when I run the tests
![screen shot 2015-11-30 at 17 19 50](https://cloud.githubusercontent.com/assets/1422605/11478761/b9e0966a-9786-11e5-81e1-9e2a4e7191be.png)

**Comment by Claus79** [comment]
btw was the change '${inherited} to the "Other Linker flags: in the MyAppUITests target' ok?

**Comment by Claus79** [comment]
all broken. Isn't there a step-by-step tutorial for running UI Tests on project using CocoaPods?

**Comment by segiddins** [comment]
¯\_(ツ)_/¯ we'd need an example project we could debug to really know for sure. I've never used Xcode's UI Testing.

**Comment by Claus79** [comment]
could you give me an email address to send it over?

**Comment by Claus79** [comment]
I have also de-integrated the project and run pod install again but I keep on getting the same error.

**Comment by Douvi** [comment]
I have the same bug since I download XCode 7.2.1...

The bundle “projectUITests” couldn’t be loaded because it is damaged or missing necessary resources. Try reinstalling the bundle.

.../projectUITests.xctest/projectUITests): Library not loaded: @rpath/libswiftAVFoundation.dylib
Referenced from: .../Debug-iphonesimulator/RSBarcodes_Swift.framework/RSBarcodes_Swift 
Reason: image not found)

**Comment by neonichu** [comment]
This issue has been inactive for a long time and will be closed because we want to move forward with a clean slate after our 1.0 release. If you believe this is still relevant, please retest with the 1.0 release candidate and comment with a reproducible project in order for this to become actionable again. Thanks!

**Comment by raptox** [comment]
issue is till relevant, just tested it with 1.0. It is really important to get this fixed for the UI tests.

**Comment by foffer** [comment]
I'm still experiencing this issue with cocoapods 1.0

**Comment by schube** [comment]
We are having the exact same problem and are looking forward for a solution. Thank you!

**Comment by rob-nash** [comment]
I'm getting this issue and I am not using Cocoapods at all.
